### ISSUE TITLE
Sensor continues to be logged after Multiple Persons Detected

### ISSUE THREAD

**Issue by jschrempp** [SISProject/SISSoftware#56]
Small bug... If Multiple Persons Detected is asserted, then all PIRs should be ignored (locked out) until an exterior door is tripped. However, we found that after MPD is asserted then No Movement Detected is asserted, the last PIR to trip before MPD is still active. What we saw in the device logs:

Bedroom 1
Multiple Persons
No Movement
Bedroom 1
No Movement
Bedroom 1

After Multiple Persons is asserted, there should be no more sensor reports or events until an exterior door is tripped. Our assumption is that once we have an MPD, we can't tell anything else about what's going on in the house; someone needs to leave the house.

**Comment by SISProject** [comment]
I agree that this is a small bug.  In fact, it is insignificant for 2 reasons:  (1) it is most unlikely to ever occur in the real world, and (2) there are no negative consequences to logging a PIR trip after an hour of NMD, even after MPD.  The reason for the MPD lockout was so that the log didn't fill up with "noise" owing to multiple persons moving about the house.

I do not necessarily agree that PIRs shouldn't be logged in this instance.  We need to think more about it.  After NMD, we do want to know when movement is again detected, and where.  The problem is that with the assumption that multiple persons are present, it isn't clear what the movement means other than the people aren't all dead.
